projects/storefrontapp-e2e-cypress/cypress/integration/vendor/digital-payments/digital-payments.e2e-spec.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  loginUser,
+  goToCheapProductDetailsPage,
+  addCheapProductToCartAndBeginCheckoutForSignedInCustomer,
+  signOut,
+} from '../../../helpers/checkout-flow';
+import {
+  my_user,
+  orderConfirmation,
+  checkoutShippingAddress,
+  checkoutDeliveryMode,
+  checkoutPaymentDetails,
+  reviewAndPlaceOrder,
+} from '../../../helpers/vendor/digital-payments/user';
+
+describe('checkout using digital-payments', () => {
```

Comment:
The new credit card is filled in a 3rd party page, with a completely different domain. Cypress.io cannot navigate across different domains, so the test script is written this way.

Reply:
Since cypress.io cannot navigate from one domain to another, the 3rd party page navigation is not possible. Hence the test script always uses a defaulted credit card.